migrator/migrations/20240805163854_circle_report.up.sql

```diff
@@ -0,0 +1,15 @@
+create table
+  "circle_report" (
+    id serial primary key,
+    user_id integer not null,
+    circle_id integer not null,
+    reason text,
```

Comment:
```suggestion
    reason varchar(255),
```
aku lebih preferer varchar, just in case frontend tidak ada validasi bisa diabuse sama user
lebih hemat memori juga

Reply:
noted